test/e2e/utils/pod.go

```diff
@@ -17,14 +17,17 @@ limitations under the License.
 package utils
 
 import (
+    "context"
+	"regexp"
 	"sync"
 
 	"github.com/onsi/ginkgo"
 
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/kubernetes/test/e2e/framework"
+    clientset "k8s.io/client-go/kubernetes"
+    "k8s.io/kubernetes/test/e2e/framework"
```

Comment:
please run `gofmt` on sources before uploading. Or, even better, let's add a CI lane to enforce this check :)